code_test.go

```diff
@@ -38,6 +39,8 @@ func TestCode(t *testing.T) {
 		assertCodeRoundTrips(t, code)
 	}
 	assertCodeRoundTrips(t, Code(999))
+	var canceledInvalid Code
+	assert.NotNil(t, canceledInvalid.UnmarshalText([]byte("code_"+strconv.Itoa(int(CodeCanceled)))))
```

Comment:
This test failed before updating minCode.

Reply:
Gotcha. Definitely see how this test would have kept us from making the mistake we did, but I'm not sure it's worth keeping - it covers the same code as the change to `assertCodeRoundTrips`, and if we make a logical mistake again we'll almost certainly make the same mistake updating this test.

Up to you, I'm just saying that I'd be comfortable without this addition.